docker/dev_env/shared_aliases.sh

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+# Place common shared utility functions here
+# Functions, not aliases, must be used. Aliases are not expanded in scripts
+# and as such will not be available in `entrypoint.sh` when the command
+# passed to the docker run is evaluated. That means that `j` below, for example,
+# wouldn't work if using `ov j`. However, as a function, it does work.
+
+# Personal aliases and utilities should go in an .ovprofile file at the monorepo root
+
+declare -A shared_aliases
```

Comment:
`declare -A` is a feature that was introduced in Bash newer than the one on macOS (version 3.2.57(1)-release). This doesn't work for me.

Reply:
Just adding a note here, we tried switching to zsh, and that worked great, except that shellcheck doesn't support it.

We only use the associative bash arrays for aliases, and I think we could easily support it another way (e.g., using Python to expand aliases). I've removed the alias support for now @AetherUnbound and we can try to accommodate that in a separate PR.